### DISCUSSION TITLE
Rendering front matter in rich-text view

### DISCUSSION THREAD

**Discussion by leekool** [mdx-editor/editor#663] [Q&A]
Is there an easy way to have the front matter displayed in rich-text view? Currently it seems only visible in source/diff mode or when opening the "Edit front matter" dialogue from the toolbar

**Comment by Thor-x86** [comment]
There is no feature for that, unfortunately. Best thing we can do is to [add frontmatter button to toolbar](https://mdxeditor.dev/editor/docs/customizing-toolbar#insertfrontmatter).
